universalizer/norm.py

```diff
@@ -406,3 +399,48 @@ def load_sssom_maps(maps) -> tuple:
     print(f"Loaded {len(cat_map)} category mappings.")
 
     return (id_map, cat_map)
+
+
+def obo_handle(old_id: str) -> str:
```

Comment:
I'd love a way to automate fixing this across the original ~1,000 Bioportal entries (because that's mostly what this is for), but for now, all IDs are written out to one of three different reports, as needed:
* IDs of unexpected format, e.g.,
```
ID
OBO:ExO_0000030
OBO:ExO_0000151
OBO:ExO_0000152
```
* IDs with remapped categories
```
Old ID	New Category
OBO:ExO_0000030	biolink:NamedThing
OBO:ExO_0000151	biolink:NamedThing
OBO:ExO_0000152	biolink:NamedThing
```
* IDs with remapped IDs
```
Old ID	New ID
OBO:ExO_0000030	EXO:0000030
OBO:ExO_0000151	EXO:0000151
OBO:ExO_0000152	EXO:0000152
```

So that last report would be most useful for finding the easily-solved exceptions, but the first report may also contain some candidates for repair.

Reply:
Remind me, why are these not correctly understood to be: `ExO:0000030`?